uwblueprint/focus-on-nature
# Feat/multi session backend

## Notion ticket link

[Supporting multi session registration](https://www.notion.so/uwblueprintexecs/Task-Board-F22-9e03bf9f7c4343f09a8a5c16dc48a12a?p=2f122d709df14568945a406933e09734&pm=c)



## Implementation description
* Creates 1 camper entity for each session that the camper registers for



## Steps to test
1. Create a POST request to `http://localhost:5000/campers/register` with the body provided in the comments below
2. You might need to edit the camp session ids to match the camp you are registering for along with any other details to work for your test case (or you can delete the entities on the db and use these values as is)

TEST WAITLIST -> CAMPER REGISTERATION 
3. Create a waitlisted camper using the sample body in the comments 
4. Make sure the session you are moving the waitlisted camper to has capacity
5. Create a POST request to `http://localhost:5000/campers/register?wId=<id of waitlisted camper>`
6. Ensure that the status of the waitlisted camper is now "Registered" and a registered Camper entity is created



## What should reviewers focus on?
* 


## Checklist
